src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,7 +11,7 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Client has been cleared!";
```

Comment:
Minor typo here, forget to include the word `list`

```suggestion
    public static final String MESSAGE_SUCCESS = "Client list has been cleared!";
```